source/string.c

```diff
@@ -25,12 +25,12 @@ struct aws_string *aws_string_new_from_c_str(struct aws_allocator *allocator, co
 }
```

Comment:
You should be able to revert the warning disables above, too!

Reply:
Good catch! I just removed them.